cmd/ilm-set.go

```diff
@@ -0,0 +1,162 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"context"
+
+	"github.com/minio/cli"
+	"github.com/minio/mc/cmd/ilm"
+	json "github.com/minio/mc/pkg/colorjson"
+	"github.com/minio/mc/pkg/probe"
+	minio "github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/lifecycle"
+	"github.com/minio/minio/pkg/console"
+)
+
+var ilmSetCmd = cli.Command{
+	Name:   "set",
+	Usage:  "modify a lifecycle configuration rule to existing (if any) rule(s) on a bucket",
```

Comment:
Does modify here mean replace the whole configuration, or just update config field changes ?

Reply:
It will just update the config fields that are changed in command line.

Reply:
> It will just update the config fields that are changed in command line.

@nitisht maybe 'update' is better than 'set' here